docs/application/native/guides/text-input/autofill-service.md

```diff
@@ -0,0 +1,335 @@
+
+        /* item 1 */
+        autofill_fill_response_item_create(&res_it_h[0]);
+        autofill_fill_response_item_set_id(res_it_h[0], "id");
+        autofill_fill_response_item_set_presentation_text(res_it_h[0], "tester1");
+        autofill_fill_response_item_set_value(res_it_h[0], "tester1");
+
+        /* Add item 1 in group 1 */
+        autofill_fill_response_group_add_item(res_group_h[0], res_it_h[0]);
+
+        /* item 2 */
+        autofill_fill_response_item_create(&res_it_h[1]);
+        autofill_fill_response_item_set_id(res_it_h[1], "password");
+
+        /* Use another presentation text due to sensitive data */
+        autofill_fill_response_item_set_presentation_text(res_it_h[1], "tester1's Password");
+        autofill_fill_response_item_set_value(res_it_h[1], "testerpw1");
+
+        /* Add item 2 in group 1 */
+        autofill_fill_response_group_add_item(res_group_h[0], res_it_h[1]);
+
+        autofill_fill_response_item_destroy(res_it_h[0]);
+        autofill_fill_response_item_destroy(res_it_h[1]);
+
+        /* Add group 1 in fill response */
+        autofill_fill_response_add_group(fill_response_h, res_group_h[0]);
+
+        autofill_fill_response_group_destroy(res_group_h[0]);
+
+        /* group 2 */
+        autofill_fill_response_group_create(&res_group_h[1]);
+
+        /* item 1 */
+        autofill_fill_response_item_create(&res_it_h[0]);
+        autofill_fill_response_item_set_id(res_it_h[0], "id");
+        autofill_fill_response_item_set_presentation_text(res_it_h[0], "tester2");
+        autofill_fill_response_item_set_value(res_it_h[0], "tester2");
+
+        /* Add item 1 in group 2 */
+        autofill_fill_response_group_add_item(res_group_h[1], res_it_h[0]);
+
+        /* item 2 */
+        autofill_fill_response_item_create(&res_it_h[1]);
+        autofill_fill_response_item_set_id(res_it_h[1], "password");
+
+        /* Use another presentation text due to sensitive data */
+        autofill_fill_response_item_set_presentation_text(res_it_h[1], "tester2's Password");
+        autofill_fill_response_item_set_value(res_it_h[1], "testerpw2");
+
+        /* Add item 2 in group 2 */
+        autofill_fill_response_group_add_item(res_group_h[1], res_it_h[1]);
+
+        autofill_fill_response_item_destroy(res_it_h[0]);
+        autofill_fill_response_item_destroy(res_it_h[1]);
+
+        autofill_fill_response_add_group(fill_response_h, res_group_h[1]);
+
+        autofill_fill_response_group_destroy(res_group_h[1]);
+
+        /* Send fill response */
+        autofill_service_send_fill_response(context_id, fill_response_h);
+
+        autofill_fill_response_destroy(fill_response_h);
+    }
+
+    bool service_app_create(void *data)
+    {
+        autofill_service_initialize();
+
+        ...
+
+        autofill_service_set_fill_requested_cb(_fill_request_received_cb, NULL);
+
+        ...
+
+        return true;
+    }
+    ```
+
+## The storage of autofill data
```

Comment:
Pleas verify the suggested change:
## Save Autofill Data

Reply:
fixed.